### PR TITLE
Improve CI and handle profiles App

### PR DESCRIPTION
**Pull request by JalilArfaoui**
This PR:

- depends upon #642, because it builds up with added webpack config
- tidies up package.json (with a lot of breaking changes, especially for Semaphore «servers»)
  - builds with sentry enabled are now called `release` => `buildVersion`
  - triggering Semantic Release is now only one script `deploy*` => `release`
  - Firefox store uploads are now `sign:firefox:*` => `upload:firefox:*`
- removes `skip-ci` from version commit messages, so that all commits appear in Semaphore, but only triggers version bump if commit message does **not** contain `chore: release`
- adds `build:profiles` and `build:profiles:production` scripts
- uploads `profiles.bundle.js` to Github releases

This is pretty surely uncomplete and has to be tested … I’ll test it on a fork with separated CI if you agree on the general approach @lutangar

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -1,16 +1,82 @@
...
+    }
+  ],
+  publish: [
+    {
+      path: '@semantic-release/exec',
+      cmd: 'yarn run buildVersion'
+    },
+    {
+      path: '@semantic-release/exec',
+      cmd: 'yarn run upload:firefox:staging'
```
Comment (lutangar) [inline release.config.js:35]: I might miss something here, can we still "release" *staging* and *proding* version without publishing *production* version?

File: webpack.profiles.config.js [hunk 2]
```diff
@@ -77,7 +60,28 @@ module.exports = function webpack(env = {}, argv = {}) {
...
         )(env)
       }),
-      new CopyWebpackPlugin(copyConfig),
+      new CopyWebpackPlugin([
+        {
+          from: 'src/assets',
+          to: defaultWebpackConfig.output.path
+        },
+        {
+          from: 'node_modules/webextension-polyfill/dist/browser-polyfill.js',
```
Comment (lutangar) [inline webpack.profiles.config.js:69]: Browser polyfill is never used in profiles app.

File: webpack.profiles.config.js [hunk 3]
```diff
@@ -77,7 +60,28 @@ module.exports = function webpack(env = {}, argv = {}) {
...
+        {
+          from: 'node_modules/typeface-lato/files/',
+          to: path.join(defaultWebpackConfig.output.path, 'fonts/')
+        },
+        {
+          from: 'node_modules/typeface-sedgwick-ave/files/',
+          to: path.join(defaultWebpackConfig.output.path, 'fonts/')
+        },
+        {
+          from: 'test/integration',
```
Comment (lutangar) [inline webpack.profiles.config.js:81]: I need to verify why I previously left it, mb not required anymore.

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -195,13 +195,13 @@ Once built each package is released on [Github](https://github.com/insitu-projec
 
 > The publication on the Chrome store may take a while -- days -- to be validated.
 
-> See the detailed `staging` deploy steps `./release.config.staging.js` in project root directory.
+> See the detailed `staging` deploy steps `./release.config.js` in project root directory.
```
Comment (lutangar) [inline docs/CONTRIBUTING.md:198]: Might add a few info on how to release on staging and how it works with Semaphore.

**Comment by JalilArfaoui** [comment]
After playing with this branch on my own fork with a dedicated Semaphore project, I’m getting confident trying this live on master (Semaphore settings will have to be adjusted prior to merge) : 

image

**Comment by JalilArfaoui** [comment]
(I didn’t let the upload to store steps in my tests though)

**Comment by lutangar** [comment]
@JalilArfaoui is this ready for review ? :)

**Comment by JalilArfaoui** [comment]
I think so !

**Pull request merged**